### PR TITLE
PMM-5804 Updating UI tests job, client instance setup

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

**Review by Sudokamikaze — APPROVED** [review]
Syntax looks good for me

**Comment by puneet0191** [comment]
I have tested it with temp job, will merge it when other changes are also ready thanks for review!

File: pmm/pmm2-ui-tests.groovy
Comment (Sudokamikaze) [inline pmm/pmm2-ui-tests.groovy:183]: Is it really needed to run this block in parallel? For example, just in case if sanity check fails, the pipeline will run till it finishes the whole stage. 

Other parts is good to me

File: pmm/pmm2-ui-tests.groovy
Comment (Sudokamikaze) [inline pmm/pmm2-ui-tests.groovy:169]: Better to be `Start PMM Server Instance` as on line 177, there's: `Start PMM Client Instance`

Comment (puneet0191) [reply pmm/pmm2-ui-tests.groovy:169]: sure will make that correction thanks.

File: pmm/pmm2-ui-tests.groovy
Comment (puneet0191) [reply pmm/pmm2-ui-tests.groovy:183]: yes, but it will still be very rare as compared to the number of jobs which will save 2-3 mins from this in parallel, but if you think it is not needed, I can move sanity Check out of this parallel block

**Review by puneet0191 — COMMENTED** [review]
just replied

File: pmm/pmm2-ui-tests.groovy
Comment (Sudokamikaze) [reply pmm/pmm2-ui-tests.groovy:183]: Got it, LGTM

**Review by Sudokamikaze — APPROVED** [review]

**Pull request merged**
